### ISSUE TITLE
Final Peer Review - vvs24

### ISSUE THREAD

**Issue by virenvshah** [emilyxia/Hospital-Information-Analysis#12]
This project analyzes the total average charges of a hospital based on the disease a patient has,demographic information about the patient, and information about the hospital.  The project primarily focuses on using regression tasks such as Linear Regression, Lasso Regression, and Random Forests to make a good prediction of this target variable.  

Things I liked about the project:

1.  The group tried several different models, and compared and contrasted the results for reach model by recording both the Train Mean Squared Error, and the Test Mean Squared Error for each model.  This made it clear to the reader, which model performed the best, along with which models were overfitting.  

2.  The group took overfitting into account in their project, and argued that due to their limited number of features, it was likely that they could be overfitting.  This was the motivation for why they used lasso regression and ridge regression. 

3.  The group analyzed the importance of various features with respect to the target variable, by analyzing the coefficients of their ridge regression classifier, and the feature importance scores of their random forest classifier.  This was a good thing to do because it helped them realize that the "Length of Stay" feature seemed to have the largest correlation with their target variable.

4.  The group's Fairness analysis was very in depth and they considered how different protected attributes varied with the target label.

Things that could be improved:

1.  The group focused a lot on model selection, but did not focus a lot on their feature representation.  I wish there was more analysis on how they represented their features.  I think they may have gotten accuracy improvements if they tried varying aspects of their feature vectors.  Another thing they could have tried is using different ways to encode categorical variables, such as using hashing instead of one hot encoding, to shrink the size of their categorical variable vectors. 

2.  During model selection, I wish there was more discussion about hyperparameter tuning.  For example, maybe they could have discussed their results from doing K-Fold cross validation to choose the optimal "max-depth" parameter for their Random Forest Regression.  

3.  During the feature importance analysis, it seemed that "length of stay" had the largest correlation with the target variable by far.  I would have liked the group to have trained only on the "length of stay" feature and recorded the test accuracy of that simple model, and compare to their overall models.  This way they would be able to show that the other features do give a definite improvement to their prediction accuracy.
